tests/base.py

```diff
@@ -127,17 +127,20 @@ def expected_metadata(self):
                 self.PRIMARY_KEYS: {"ID"},
                 self.REPLICATION_METHOD: self.INCREMENTAL,
                 self.REPLICATION_KEYS: {"ModifiedDate"},
-            } 
+            }
         }
 
-    def streams_to_select(self):
+    def streams_to_select(self, is_discovery=False):
         # events: there are 5 events and the API call window is of 10 minutes
         #   so there will be a lot of API calls for every test
         # list_subscriber: as the API window is of 1 day, the tests took
         #   30 minutes to run 3 tests, the test run time will be increased
         #   when all the tests are combined
         # subscriber: it is the child stream of 'list_subscriber'
-        return set(self.expected_metadata().keys()) - {'event', 'list_subscriber', 'subscriber'}
+        if is_discovery:
+            return set(self.expected_metadata().keys())
+
+        return set(self.expected_metadata().keys()) - {'event', 'list_subscriber', 'subscriber', 'data_extension.my_test'}
```

Comment:
Did you investigate why this stream is now failing? Is this a problem with test data, or a problem with the tap? When we lose coverage on a stream we need to root cause the issue. If the root cause cannot be easily determined then a backlog ticket  should be created to investigate the issue.